gwpy/table/filter.py

```diff
@@ -251,6 +253,6 @@ def filter_table(table, *column_filters):
     """
     keep = numpy.ones(len(table), dtype=bool)
     for name, op_func, operand in parse_column_filters(*column_filters):
-        col = table[name].view(numpy.ndarray)
+        col = table[name]
```

Comment:
Removing the `view` is required to ensure that the `op_func` receives a formatted `Column` object (or a `Table` slice), rather than just an array; this in turn is a requirement of the new documented functionality.

Reply:
Ok, thanks